packages/core/contracts/optimism/external/openzeppelin/contracts/access/Ownable.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
```

Comment:
If we have the ability to whitelist directories for the OVM, why can't we just whitelist the OZ node modules directories/contracts that we want rather than adding a second set of OZ contracts in the repo?

Reply:
Great point, I refactored a bit will tag you

Reply:
Done, no new contracts are added in this PR now